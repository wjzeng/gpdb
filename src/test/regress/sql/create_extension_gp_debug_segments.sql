--
-- Tests for return correct error from qe when create extension error
-- The issue: https://github.com/greenplum-db/gpdb/issues/11304
--

--start_ignore
drop extension if exists gp_debug_numsegments;
create extension if not exists gp_inject_fault;
--end_ignore

select gp_inject_fault('create_function_fail', 'error', 2);
create extension gp_debug_numsegments;

select gp_inject_fault('create_function_fail', 'reset', 2);

