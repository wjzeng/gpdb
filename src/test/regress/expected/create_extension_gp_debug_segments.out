--
-- Tests for return correct error from qe when create extension error
-- The issue: https://github.com/greenplum-db/gpdb/issues/11304
--
--start_ignore
drop extension if exists gp_debug_numsegments;
NOTICE:  extension "gp_debug_numsegments" does not exist, skipping
create extension if not exists gp_inject_fault;
--end_ignore
select gp_inject_fault('create_function_fail', 'error', 2);
 gp_inject_fault 
-----------------
 Success:
(1 row)

create extension gp_debug_numsegments;
ERROR:  fault triggered, fault name:'create_function_fail' fault type:'error'  (seg0 192.168.106.132:7002 pid=153052)
select gp_inject_fault('create_function_fail', 'reset', 2);
 gp_inject_fault 
-----------------
 Success:
(1 row)

